src/App/App.js

```diff
@@ -0,0 +1,12 @@
+import './App.scss';
+import { Homepage } from '../Homepage/Homepage'
+
+function App() {
```

Comment:
We might want to turn this into a class component since that's how app is _usually_ structured, but we could also revisit later if we feel there is a need to change it from a functional component.